docs/user/manual.adoc

```diff
@@ -66,6 +66,7 @@ To disable this notification put the following to `settings.json`
 The server binary is stored in:
 
 * Linux: `~/.config/Code/User/globalStorage/matklad.rust-analyzer`
+* Linux (Remote, such as WSL): `~/.vscode-server/data/User/globalStorage/matklad.rust-analyzer`
```

Comment:
```suggestion
* Linux (remote and WSL): `~/.vscode-server/data/User/globalStorage/matklad.rust-analyzer`
```

Reply:
I think WSL and SSH are both considered remote.